flink-python/pyflink/shell.py

```diff
@@ -84,7 +84,7 @@
     *         os.remove(sink_path)
     *     else:
     *         shutil.rmtree(sink_path)
-    * bt_env.exec_env().set_parallelism(1)
+    * b_env.set_parallelism(1)
```

Comment:
I think we can add "b_env" at the beginning of the example to tell the users what variable is predefined. "Use the 'bt_env' variable" -> "Use the 'b_env', 'bt_env' variable"